res_partner_fiscal_position/view/res_country.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="res_partner_fiscal_position_form">
+            <field name="name">res.partner.fiscal.position.form</field>
+            <field name="model">res.country</field>
+            <field name="inherit_id" ref="base.view_country_form"/>
+            <field name="arch" type="xml">
+                <field name="code" position="after">
+                    <field name="property_account_position" domain="[('company_id.user_ids', '=', uid)]"   
```

Comment:
If I remember correctly, the domain prevents top level company users from selecting fiscal positions from companies down in the hierarchy, which would lead to problems. Can you document this in an xml comment above the record element?